src/components/Lesson/InstructionButton.js

```diff
@@ -1,13 +1,12 @@
 import React, {PropTypes} from 'react';
-import withStyles from 'isomorphic-style-loader/lib/withStyles';
-import styles from './InstructionButton.scss';
 import Button from 'react-bootstrap/lib/Button';
 import LinkContainer from 'react-router-bootstrap/lib/LinkContainer';
 
-const InstructionButton = ({buttonPath, buttonText, bsStyle, bsSize}) => {
+const InstructionButton = ({buttonPath, buttonText, className, bsSize}) => {
```

Comment:
Bare et forslag:
```
const InstructionButton = ({buttonPath, buttonText, ...buttonProps}) =>
```
og
```
<Button componentClass='div' {...{...buttonProps, bsStyle}}>
```
og så fjerne `bsSize` og `className` fra proptypes... poenget her er at InstructionButton tillater da å sende videre props til button uten at InstructionButton trenger å vite om det. Men legg merke til at bsStyle kommer sist, sånn at ingen kan overstyre denne.

Som sagt, bare et forslag, legg til om du føler for det.

Reply:
Hm, jeg synes det er mer oversiktlig sånn det er nå